### PR TITLE
Readmes rach java2

### PR DESCRIPTION
**Pull request by Rachael-E**
Next batch of readmes, @hud10837 could you take a first look please?

### CODE REVIEW

File: java/read-geopackage/README.md [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
+![Image of read GeoPackage](read-geopackage.png)
+
+## Use case
+
+The OGC GeoPackage specification defines an open standard for sharing raster and vector data. You may want to use GeoPackage files to support file-based sharing of geographic data.
 
 ## How to use the sample
-The `Layer`s in the `GeoPackage` are shown in a drawer on the left hand side of the screen. Click on a `Layer` to add or remove it from map.
+
+When the sample loads, the feature tables and rasters from the GeoPackage will be shown on the map.
```
Comment (hud10837) [inline java/read-geopackage/README.md]: It looks like there is a menu on the side here to select layers to add, perhaps this should be mentioned here?

File: java/read-geopackage/README.md [hunk 2]
```diff
@@ -1,25 +1,32 @@
...
 
 ## How it works
-1. Create a `GeoPackage` using a path a the local GeoPackage file.
-1. Load the `GeoPackage` with `loadAsync()`.
-1. Use `GeoPackage.getGeoPackageRasters()` to get each `GeoPackageRaster` and use them to make `RasterLayer`s.
-1. Use `GeoPackage.getGeoPackageFeatureTables()` to get each `GeoPackageFeatureTable` and use them to make `FeatureLayer`s.
-1. Based on user input, add or remove these `Layer`s from the `ArcGISMap`'s operationalLayers.
+
+1. Open the `GeoPackage` with a path and load it using `GeoPackage.loadAsync`.
+2. Iterate through available rasters, exposed by `geopackage.getGeoPackageRasters`.
```
Comment (hud10837) [inline java/read-geopackage/README.md]: I think we are using class names over instance names?
```suggestion
2. Iterate through available rasters, exposed by `GeoPackage.getGeoPackageRasters`.
```

File: java/read-geopackage/README.md [hunk 3]
```diff
@@ -1,25 +1,32 @@
...
+    * For each raster, create a raster layer using `new Rasterlayer(geopackageRaster)`, then add it to the map.
+3. Iterate through available feature tables, exposed by `geopackage.getGeoPackageFeatureTables`.
```
Comment (hud10837) [inline java/read-geopackage/README.md]: same as above
```suggestion
3. Iterate through available feature tables, exposed by `GeoPackage.getGeoPackageFeatureTables`.
```

File: java/read-geopackage/README.md [hunk 4]
```diff
@@ -1,25 +1,32 @@
...
+    * For each feature table, create a feature layer using `new FeatureLayer(geopackageFeatureTable)`, then add it to the map.
 
 ## Relevant API
-* FeatureLayer
+
 * GeoPackage
+* GeoPackage.GeoPackageFeatureTables
```
Comment (hud10837) [inline java/read-geopackage/README.md]: Delete the method signatures in the API section
```suggestion
```

File: java/read-geopackage/README.md [hunk 5]
```diff
@@ -1,25 +1,32 @@
...
+* GeoPackage.GeoPackageRasters
```
Comment (hud10837) [inline java/read-geopackage/README.md]:
```suggestion
```

File: java/scene-layer/README.md [hunk 6]
```diff
@@ -1,20 +1,35 @@
...
+
+When launched, this sample displays a scene service with an `ArcGISSceneLayer`. Pan and zoom to explore the scene.
 
 ## How it works
-1. Create an `ArcGISScene` and set a `Basemap` with `ArcGISScene.setBasemap()`.
-1. Create a `SceneView` and set the scene to the view with `SceneView.setScene(...)`. 
-1. Create a `SceneLayer` and add it to the scene as an operational layer with `ArcGISScene.getOperationalLayers().add(sceneLayer)`.
+
+1. Create an `ArcGISScene` and set its `Basemap` with `ArcGISScene.setBasemap()`.
+1. Create a `SceneView` and set the scene to the view with `SceneView.setArcGISScene(scene)`.
```
Comment (hud10837) [inline java/scene-layer/README.md]: Looks like this method is different
```suggestion
1. Create a `SceneView` and set the scene to the view with `SceneView.setScene(scene)`.
```

File: java/scene-layer/README.md [hunk 7]
```diff
@@ -1,20 +1,35 @@
...
+
+## About the data
+
+The scene launches with a northward view of the city of Brest, France. A 3D scene layer representing buildings (some textured) is used as an example.
 
 ## Relevant API
+
 * ArcGISScene
+* ArcGISSceneLayer
+* ArcGISTiledElevationSource
```
Comment (hud10837) [inline java/scene-layer/README.md]: As with kotlin, I don't see `ArcGISTiledElevationSource` used in this sample.

File: java/read-geopackage/README.md (hunk 2 above)
Comment (Rachael-E) [reply java/read-geopackage/README.md]: I'll keep this as a variable name since the geopackage was "instantiated" above

**Comment by Rachael-E** [comment]
thanks @hud10837 for keeping me straight with these! could you take another look please?

**Comment by Rachael-E** [comment]
thanks! @sorenoid could you do a 2nd review please?

File: java/scene-layer/README.md [hunk 8]
```diff
@@ -1,20 +1,34 @@
...
+
+When launched, this sample displays a scene service with an `ArcGISSceneLayer`. Pan and zoom to explore the scene.
 
 ## How it works
-1. Create an `ArcGISScene` and set a `Basemap` with `ArcGISScene.setBasemap()`.
-1. Create a `SceneView` and set the scene to the view with `SceneView.setScene(...)`. 
-1. Create a `SceneLayer` and add it to the scene as an operational layer with `ArcGISScene.getOperationalLayers().add(sceneLayer)`.
+
+1. Create an `ArcGISScene` and set its `Basemap` with `ArcGISScene.setBasemap()`.
+1. Create a `SceneView` and set the scene to the view with `SceneView.scene(scene)`.
```
Comment (sorenoid) [inline java/scene-layer/README.md]: use java setter :) 
```suggestion
1. Create a `SceneView` and set the scene to the view with `sceneView.setScene(scene)`.
```

**Comment by Rachael-E** [comment]
@sorenoid thanks for those! Changes made, merging now.

**Pull request merged**